lib/landable/traffic/user_tracker.rb

```diff
@@ -40,6 +40,8 @@ def record_page_view
 
       def save
         record_page_view
+        p = record_page_view
+        EventPublisher.new(p).publish
```

Comment:
This is inefficient when a publisher isn't configured. Its going to constantly create objects which GC has to collect, when publish is simply going to return when amqp_enabled is false. Don't create objects unnecessarily.